MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include *.txt
+include README.rst
```

Comment:
They are included by the *.txt line above.

I don't know about recursive inclusion of MANIFEST.